packages/web-ui/src/sections/Document/Sidebar/Files/index.tsx

```diff
@@ -211,7 +211,9 @@ export function FilesTree({
           confirm={{
             isConfirming: isDestroying,
             label: deletingFolder ? 'Delete folder' : 'Delete prompt',
-            description: `Deleting this ${deletingFolder ? 'folder' : 'prompt'} will also delete all the prompts it contains. This action cannot be undone.`,
+            description: deletingFolder
+              ? 'Deleting this folder will also delete all the prompts it contains. This action cannot be undone.'
+              : 'Deleting this prompt might affect your production app and other prompts referencing it. This action cannot be undone.',
```

Comment:
Hmm I don't get the second warning. Deleting anything does not affect production untill you merge the commit no? Also it could be undone no? Documents are soft deleted

Reply:
If you delete some document and you merge it without checking reference will break production